main.go

```diff
@@ -373,6 +379,50 @@ func main() {
 		}
 	}
 
+	if tlsEncryption {
+		sslOpts := &gocql.SslOptions{
+			Config:&tls.Config{
+				ServerName: serverName,
+			},
+			EnableHostVerification: hostVerification,
+		}
+
+		if caCertFile != "" {
+			if _, err := os.Stat(caCertFile); err != nil {
+				log.Fatal(err)
+			}
+			sslOpts.CaPath = caCertFile
+		}
+
+		if clientKeyFile != "" {
+			if _, err := os.Stat(clientKeyFile); err != nil {
+				log.Fatal(err)
+			}
+			sslOpts.KeyPath = clientKeyFile
+		}
+
+		if clientCertFile != "" {
+			if _, err := os.Stat(clientCertFile); err != nil {
+				log.Fatal(err)
+			}
+			sslOpts.CertPath = clientCertFile
+		}
+
+		if clientKeyFile != "" || clientCertFile != "" {
+			if clientKeyFile == "" || clientCertFile == "" {
+				log.Fatal("ssl-client-key-file and ssl-client-cert-file should be provided side by side")
```

Comment:
I think that this error message could be formulated better. "side by side" suggests to me that both options should be near each other in some way (next to each other as commandline arguments? or maybe key and cert file need to be in the same catalogue?).

How about doing two checks and splitting the error message like this:
```go
if clientKeyFile != "" && clientCertFile == "" {
    log.Fatal("ssl-client-cert-file is required when ssl-client-key-file is provided")
}
if clientCertFile != "" && clientKeyFile == "" {
    log.Fatal("ssl-client-key-file is required when ssl-client-cert-file is provided")
}
```